RDAPI-20494 Creation of new apigateway with yaml factory option (#1701)
Axway/axway-open-docs: content/en/docs/apim_yamles/yamles_packaging_deployment.md

"description": "Learn how to package and deploy a YAML configuration to the API Gateway runtime."
}

> {{< alert title="Note">}}When a YAML configuration is deployed to an API Gateway group, it is still possible to switch it back and deploy an XML federated configuration to that same API Gateway group. When an API Gateway instance is first created, an XML federated configuration or a YAML configuration can be deployed in it.{{< /alert>}}

Reviewer: ```suggestion
{{< alert title="Note">}}When a YAML configuration is deployed to an API Gateway group, it is still possible to switch it back and deploy an XML federated configuration to that same API Gateway group. When an API Gateway instance is first created, an XML federated or YAML configuration is deployed with it.{{< /alert>}}
```
